apps/storefront/src/components/table/B3NoData.tsx

```diff
@@ -30,11 +31,20 @@ export default function B3NoData({
   text,
   backgroundColor,
   minHeight,
+  showBorder = false,
   isLoading = false,
 }: B3NoDataProps) {
   const b3Lang = useB3Lang();
   return (
-    <NoDataContainer backgroundColor={backgroundColor} minHeight={minHeight}>
+    <NoDataContainer
+      style={{
+        boxShadow: showBorder
```

Comment:
Not a huge fan of these kind of booleans, it might be better to make this part of the theme, something like `panelShadow`. Just give it a name and toggle it as part of theme.

```tsx
// theme.tsx

declare module '@mui/material' {
  interface Theme {
    styles: {
      panelShadow: string;
    };
  }
  interface ThemeOptions {
    styles: {
      panelShadow: string;
    };
  }
}

// ...

createTheme(
  {
    styles: {
      panelShadow:
        '0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%)',
    },
	// ...
```

and then in here:

```tsx
const theme = useTheme();

return (
  <NoDataContainer
    backgroundColor={backgroundColor}
    style={{ boxShadow: theme.styles.panelShadow }}
```